apps/benchmark/src/implementations/inline-styles/View.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+const compose = (s1: React.CSSProperties, s2: React.CSSProperties) => {
+  if (s1 && s2) {
+    return { ...s1, ...s2 };
+  } else {
+    return s1 || s2;
+  }
+};
+
+class View extends React.Component<{ style: React.CSSProperties }> {
```

Comment:
nit: Let's use function components

Reply:
Done in [4ed98dcb](https://github.com/microsoft/griffel/pull/132/commits/4ed98dcbe09cfc92ebbf8ba627bcbff2301fa426)